src/views/CreateTender/CreateTender.vue

```diff
@@ -0,0 +1,361 @@
+<template>
+  <div class="tender-form-header-container">
+
+  </div>
+  <div  style="margin-top: -155px;width: 60%;
+    margin-left: 20%;">
+    <div class="tender-form-title">
+      <h1> Create New Tender </h1>
+    </div>
+
+    <div class="tender-form">
+      <h2>Contractor</h2>
+      <div class="input-container">
+        <label for="nameId"><span class="asterix">*</span>Official name  </label>
+        <input id="nameId" class="tender-input" v-model="name" placeholder="Official name" />
+      </div>
+      <div class="input-container">
+        <label><span class="asterix">*</span>National registration number</label>
+        <input class="tender-input" v-model="nationalRegNumber" placeholder="National registration number" />
+      </div>
+      <div class="input-container">
+        <label><span class="asterix">*</span>Country</label>
+        <select v-model="countryId">
+          <option :value="null">Select</option>
+          <option v-for="countryToSelect in countries" :key="countryToSelect" :value="countryToSelect.countryId">{{countryToSelect.name}}</option>
+        </select>
+      </div>
+      <div class="input-container">
+        <label>City/Town</label>
+        <input class="tender-input" v-model="townCity" placeholder="City/Town" />
+      </div>
+
+      <h2>Contract person</h2>
+      <div class="input-container">
+        <label><span class="asterix">*</span>Name</label>
+        <input class="tender-input" v-model="contactPersonName" placeholder="Name" />
+      </div>
+      <div class="input-container">
+        <label><span class="asterix">*</span>Surname</label>
+        <input class="tender-input" v-model="contactPersonSurname" placeholder="Surname" />
+      </div>
+      <div class="input-container">
+        <label><span class="asterix">*</span>Phone number</label>
+        <input class="tender-input" v-model="phoneNumber" placeholder="Phone number" />
+      </div>
+      <h2>Subject matter of procurement</h2>
+      <div class="input-container">
+        <label><span class="asterix">*</span>CVP code</label>
+        <select v-model="cvpCodeId">
+          <option :value="null">Select</option>
+          <option v-for="cvpCodeSelect in cvpCodes" :key="cvpCodeSelect" :value="cvpCodeSelect.id">{{cvpCodeSelect.name}}</option>
+        </select>
+      </div>
+      <div class="input-container">
+        <label><span class="asterix">*</span>Type of tender</label>
+        <select v-model="tenderTypeId">
+          <option :value="null">Select</option>
+          <option v-for="tenderType in tenderTypes" :key="tenderType" :value="tenderType.id">{{tenderType.name}}</option>
+        </select>
+      </div>
+      <div class="input-container">
```

Comment:
make sense to create a generic input component wrapper and pass only props there as I told Ruslan yesterday